package.json

```
@@ -34,7 +34,7 @@
   },
   "devDependencies": {
     "@tsconfig/node-lts": "^18.12.3",
-    "turbo": "^1.10.12",
+    "turbo": "^2.0.4",
```

let's keep the version pinned
```suggestion
    "turbo": "2.0.4",
```